app/components/request-icon.js

```diff
@@ -6,6 +6,7 @@ const eventToIcon = {
   pull_request: 'pullrequest',
   cron: 'cronjobs',
   api: 'api',
+  release: 'api',
```

Comment:
As for right now - yes, since we do not have a specific icon for release events.

Reply:
@murtaza-swati Can you please provide what should be used as the icon?

Reply:
I have asked for that from Stan - Maybe UI engineer input would be required.
It can take days that we would get this. so let's not wait for this I think we can go with this for now. we can change this later.